discord.h

```diff
@@ -152,6 +152,11 @@ typedef void (*discord_on_interaction)(
   struct discord *client, const struct discord_interaction *interaction);
 /** @} */
 
+typedef void (*discord_on_guild)(struct discord *client,
```

Comment:
Can you move this to the `DiscordCallbacksGuild` doxygen group? You can find it [here](https://github.com/cee-studio/orca/blob/multiplex/discord.h#L111)